che-theia-plugins.yaml

```diff
@@ -768,3 +768,11 @@ plugins:
     repository:
       url: 'https://github.com/BroadcomMFD/cobol-control-flow'
     extension: https://open-vsx.org/api/BroadcomMFD/ccf/0.4.0/file/BroadcomMFD.ccf-0.4.0.vsix
+  - repository:
+      url: 'https://github.com/randomchance/vscode-logstash-configuration-syntax'
+      revision: '0.0.4'
+    extension: 'https://open-vsx.org/api/RandomChance/logstash/0.0.4/file/RandomChance.logstash-0.0.4.vsix'
+  - repository:
+      url: 'https://github.com/fbaligand/vscode-logstash-editor'
+      revision: '1.1.3'
```

Comment:
Probably it should be `v1.1.3` https://github.com/fbaligand/vscode-logstash-editor/tree/v1.1.3

Reply:
You are right, it's `v1.1.3` instead `1.1.3`